api/features/workflows/core/models.py

```diff
@@ -90,6 +97,28 @@ def commit(self, committed_by: "FFAdminUser"):
         self.committed_by = committed_by
         self.save()
 
+        message = CHANGE_REQUEST_COMMITED_MESSAGE % self.title
```

Comment:
This could be done using an `AFTER_SAVE` hook, right?

Reply:
yeah